src/app/dataset-view/additional-components/linage-component/linage-component.ts

```diff
@@ -18,7 +18,7 @@ export class LinageComponent {
         tableSource: any;
         isResultQuantity: boolean;
         isClickableRow: boolean;
-        pageInfo: PageInfoInterface;
+        pageInfo: SearchResultConnection;
```

Comment:
This field looked odd to me, so I started to look how it's used, but I couldn't find a single use of `tableData` field as a whole. 

It's declared as input but I don't see it set in `dataset.component.html` - the only place that instantiates this component.